bloodytools/simulation_objects/simulation_objects.py

```diff
@@ -653,19 +655,16 @@ def simulate(self) -> bool:
             .format(self.filename)
           )
         else:
-          self.filename = str(uuid.uuid4()) + ".simc"
+          # temporary file names
+          self.uuid = str(uuid.uuid4())
+          self.filename = "{}.simc".format(self.uuid)
+          json_filename = "{}.json".format(self.uuid)
 
-          # if somehow the random naming function created the same name twice
-          if os.path.isfile(self.filename):
-            self.logger.info(
-              "Somehow the random filename generator generated a name ('{}') that is already on use.".format(
-                self.filename
-              )
-            )
-            self.filename = str(uuid.uuid4()) + ".simc"
           # write arguments to file
           with open(self.filename, "w") as f:
             # write the equal values to file
+            f.write("json={}\n".format(json_filename))
+            f.write("profileset_metric={}\n".format(",".join(["dps"])))
```

Comment:
I have no clue
1) what a `profileset_metric` is
2) why you wrote that line as you did? support for extraction of more information, separated by comma?

Reply:
Well SimC has this metric concept, which is mostly a reference to some collected data sample_data object.
dps, dpse, hps, hpse, dtps, tmi, etc.

Now with normal sims, you get all of them collected (although with different amount of detail, depending on the statistics_level).

With profileset because of the amount of data, only dps is collected by default.

The idea would be to add eg. dps, hps, tmi here. For now I reverted it to just 'dps'.
So basically this line has no effect for now.